python/ray/autoscaler/kubernetes/operator_configs/example_cluster_resource.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: cluster.ray.io/v1
+kind: RayCluster
+metadata:
+  name: test-cluster
+spec:
+  targetUtilizationFraction: 0.8
+  idleTimeoutMinutes: 5
+  headPodType: head-node
+  workerDefaultPodType: worker-nodes
+  podTypes:
+  - name: head-node
+    podConfig:
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        generateName: test-cluster-ray-head-
+        labels:
+          component: ray-head
+      spec:
+        restartPolicy: Never
+
+        # This volume allocates shared memory for Ray to use for its plasma
+        # object store. If you do not provide this, Ray will fall back to
+        # /tmp which cause slowdowns if is not a shared memory volume.
+        volumes:
+        - name: dshm
+          emptyDir:
+            medium: Memory
+        containers:
+        - name: ray-node
+          imagePullPolicy: Always
+          image: rayproject/ray:nightly
+          # Do not change this command - it keeps the pod alive until it is
+          # explicitly killed.
+          command: ["/bin/bash", "-c", "--"]
+          args: ['trap : TERM INT; sleep infinity & wait;']
+          ports:
+          - containerPort: 6379 # Redis port.
+          - containerPort: 6380 # Redis port.
+          - containerPort: 6381 # Redis port.
+          - containerPort: 12345 # Ray internal communication.
+          - containerPort: 12346 # Ray internal communication.
+
+          # This volume allocates shared memory for Ray to use for its plasma
+          # object store. If you do not provide this, Ray will fall back to
+          # /tmp which cause slowdowns if is not a shared memory volume.
+          volumeMounts:
+          - mountPath: /dev/shm
+            name: dshm
+          resources:
+            requests:
+              cpu: 1000m
+              memory: 512Mi
```

Comment:
I wonder if it's okay for memory request to be different from memory limit.

Reply:
Good question. It's definitely a bit weird to have as the example